src/opds_dumper.cpp

```diff
@@ -341,34 +341,33 @@ std::string OPDSDumper::noJSDownloadLinks(const std::string& bookId) const
   );
 }
 
-std::string OPDSDumper::noJSData(kiwix::Filter filter) const
+std::string OPDSDumper::dumpPlainHTML(const std::vector<std::string>& bookIds, const std::string& query,
+                                const std::string& category, const std::string& language) const
```

Comment:
With `HTMLDumper` now being separate from `OPDSDumper` the difference in the signatures of the dump functions doesn't matter that much. The version taking a single `Filter` parameter was better.

Reply:
Done.